internal/build/main.arrai

```diff
@@ -0,0 +1,26 @@
+\bytes
```

Comment:
Nit: It would be nice to have a comment in this file to explain what it's for. It's pretty trippy.